pkgs/tools/security/rustscan/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
```

```suggestion
{ lib
```

I saw that you are only using `stdenv` for meta. `lib` is more lightweight if only needed for that purpose.